app/store.jsx

```diff
@@ -3,6 +3,7 @@ import { composeWithDevTools } from 'redux-devtools-extension'
 import rootReducer from './reducers'
 import createLogger from 'redux-logger'
 import thunkMiddleware from 'redux-thunk'
+//import promise from 'redux-promise'
 
 import {whoami} from './reducers/auth'
```

Comment:
If you aren't using this anymore just get rid of the commented line and be sure to npm uninstall --save redux-promise to update your package.json!